.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/charts/pmm/ @gen1us2k @atymchuk @denisok @spron-in
+/charts/pmm/ @BupycHuk @ademidoff @spron-in
```

Comment:
let's use teams instead of exact persons. percona/pmm-review-be should be the best option here